src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -753,7 +753,6 @@
           "description": "The set of results contained in an SARIF log. The results array can be omitted when a run is solely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
           "type": "array",
           "minItems": 0,
-          "uniqueItems": true,
```

Comment:
Note that in circumstances where a caller might want unique items, they can get them in O(N) with very little code e.g. by passing the comparer to https://msdn.microsoft.com/en-us/library/bb338049(v=vs.110).aspx